packages/app/src/settings/device/NotificationSettings.vue

```diff
@@ -0,0 +1,154 @@
+<template>
+  <SettingsSection>
+    <template #title>
+      {{ t('settingsPage.notifications.title') }}
+    </template>
+    <template #description>
+      {{ t('settingsPage.notifications.description') }}
+    </template>
+    <div class="divide-y border rounded divide-gray-50 border-gray-100 px-[16px]">
+      <div
+        v-for="({id, title}) in switches"
+        :key="id"
+        class="py-[16px]"
+      >
+        <h4 class="flex text-gray-800 text-[16px] leading-[24px] items-center">
+          <label :for="id">{{ title }}</label>
+          <Switch
+            class="mx-[8px]"
+            :value="props.gql.localSettings?.preferences[id] ?? false"
+            :name="id"
+            @update="(value) => updatePref(value, id)"
+          />
+          {{ gql.localSettings }}
+        </h4>
+      </div>
+      <div class="py-[16px]">
+        <h4 class="flex text-gray-800 text-[16px] leading-[24px] items-center">
+          {{ t('settingsPage.notifications.notifyMeWhenRunCompletes') }}
+        </h4>
+        <div class="pt-[5px]">
+          <Checkbox
+            v-for="({id, label}) in statuses"
+            :id="id"
+            :key="id"
+            v-model="listRef"
+          >
+            <template #label>
+              <label
+                :for="id"
+                class="text-sm text-gray-900"
+              >
+                {{ label }}
+              </label>
+            </template>
+          </Checkbox>
+        </div>
+      </div>
+      <div class="py-[16px]">
+        <Button
+          variant="white"
+          size="32"
+          @click="showTestNotification"
+        >
+          {{ t('settingsPage.notifications.sendTestNotification') }}
+        </Button>
+        <i18n-t
+          tag="p"
+          scope="global"
+          keypath="settingsPage.notifications.notReceivingNotifications"
+          class="pt-[15px] text-gray-600 text-sm font-normal"
+        >
+          <ExternalLink href="#">
```

Comment:
We will add the link to the docs for this after https://github.com/cypress-io/cypress-documentation/issues/5280 is completed

Reply:
Maybe better to pull out a `computed` to generate the `getUrlWithParams` with a `#` base URL so we can at least put the `TODO` alongside the needed work. Things like this inside the template tend to get lost in the noise